travis/stashcache-cache-config/10-stash-cache.conf

```diff
@@ -0,0 +1,4 @@
+[program:stash-cache]
```

Comment:
Why do we have to add this file? Isn't this already included in the container image?

Reply:
Yes but it it includes the authenticated and I did not want to get the results of errors of supervisrod.

Reply:
Ah ok, could you add a comment to the supervisord file that you're adding explaining the reasoning?

Reply:
I managed to get it working removing this.